ang/civicase/case/search/directives/search.directive.js

```diff
@@ -344,6 +323,30 @@
         });
     }
 
+    /**
+     * Sets the Case Types Based on Case Type Category
+     */
+    function setCaseTypesBasedOnCategory () {
+      var filteredCaseTypes = caseTypes;
+
+      if ($scope.caseTypeCategory) {
+        var caseTypeCategory = _.find(caseTypeCategories, function (category) {
+          return category.name.toLowerCase() === $scope.caseTypeCategory.toLowerCase();
+        });
+      }
+
+      if (caseTypeCategory) {
+        filteredCaseTypes = _.chain(caseTypes)
```

Comment:
Removed `chain`. But for all array related stuff we are using lodash, and I think its better to stick to that because of consistency. That is what we followed in civihr too. But please let me know if you think otherwise.

Reply:
It's good. Will keep it in mind.